docs/storybook/src/components/MessageCenterField.stories.tsx

```diff
@@ -4,7 +4,45 @@
  *--------------------------------------------------------------------------------------------*/
 import type { Decorator, Meta, StoryObj } from "@storybook/react";
 import { AppUiDecorator, InitializerDecorator } from "../Decorators";
-import { MessageCenterField } from "@itwin/appui-react/src/appui-react/statusfields/MessageCenter";
+import { MessageManager, MessageCenterField } from "@itwin/appui-react";
+import {
+  NotifyMessageDetails,
+  OutputMessagePriority,
+} from "@itwin/core-frontend";
+import { DropdownButton, MenuItem } from "@itwin/itwinui-react";
+
+const menuItems = [
```

Comment:
Looks great. Now let's add more stories:
- Empty (instead of Basic) - renders a component w/o any messages
- Information (renders a component with a couple of information messages)
- Error (renders a component with a couple of error messages)
- Detailed (I think there could be an advanced message, i.e. w/ sublabel, let's showcase that)

Additionally, please update `onClick` handlers (this will help us test more scenarios and see if everything is working correctly):
- `Add info` should add w/o clearing messages
- `Add error` should add w/o clearing messages
- `Clear` should clear all messages

Reply:
- Looks like detailed message styling is not right (it renders message and details in two columns). You can see how one looks in test-app -> Messages Tool -> Sticky Message
- Stories need some refinement: Notification, Error, Detailed should have some messages by default once the story is opened.
- Buttons to `Add Info`, `Add Error`, `Add Detailed`, `Clear` should all be visible (otherwise we can't fully test what happens when new messages of different types are added). And it should be possible to add multiple messages, I can only add 1 now. (you can actually create a separate story w/ these buttons, other stories don't need the buttons then).

Reply:
I.e. here's how detailed message looks in the test-app currently:
![Screenshot 2024-04-05 at 16 37 03](https://github.com/iTwin/appui/assets/10091419/9e068874-2ff4-49db-b8ea-3482c6d5e3c7)